src/docfx/build/legacy/metadata/LegacyMetadata.cs

```diff
@@ -88,7 +88,7 @@ public static JObject GenerataCommonMetadata(JObject metadata, Docset docset)
                     : null,
                 ["update_at"] = pageModel.UpdatedAt.ToString(culture.DateTimeFormat.ShortDatePattern, culture),
             };
-            rawMetadata["author"] = pageModel.Author?.ProfileUrl.Substring(pageModel.Author.ProfileUrl.LastIndexOf('/'));
+            rawMetadata["author"] = pageModel.Author?.ProfileUrl.Substring(pageModel.Author.ProfileUrl.LastIndexOf('/') + 1);
```

Comment:
can we have a name field in GitUser? ProfileUser does not always ends with author name.

Reply:
really?
interesting!

Reply:
We should not depend on the schema of profile URL.